src/core/sequence.js

```diff
@@ -100,6 +153,7 @@ Sequence.prototype.root = function(){
 };
 
 // Get a string indicating the sequence type chain represented by sources.
+// TODO: This probably doesn't need to be in the prototype, honestly
```

Comment:
At some point I want to move it to a function that just gets a type string from whatever value. The only place I use this is for error messages and most of them have to account for other types also.